src/main.cpp

```diff
@@ -2596,7 +2596,7 @@ void static UpdateTip(CBlockIndex *pindexNew, const CChainParams& chainParams) {
             ThresholdState state = checker.GetStateFor(pindex, chainParams.GetConsensus(), warningcache[bit]);
             if (state == THRESHOLD_ACTIVE || state == THRESHOLD_LOCKED_IN) {
                 if (state == THRESHOLD_ACTIVE) {
-                    strMiscWarning = strprintf(_("Warning: unknown new rules activated (versionbit %i)"), bit);
+                    strMiscWarning = strprintf("Warning: unknown new rules activated (versionbit %i)", bit);
```

Comment:
Removing translation here results in this message being shown untranslated in the UI - try it now on testnet.